cpp/include/Ice/Communicator.h

```diff
@@ -142,11 +142,17 @@ namespace Ice
          * communicator as is used by the adapter. Attempts to create a named object adapter for which no configuration
          * can be found raise InitializationException.
          * @param name The object adapter name.
-         * @param serverAuthenticationOptions The SSL configuration properties for server connections.
+         * @param serverAuthenticationOptions The %SSL configuration properties for server connections.
```

Comment:
What does `%SSL` mean here?

Reply:
`%` avoids auto-linking a symbol, otherwise `SSL` will be a link to namespace SSL.